### PR TITLE
LPS-68854 Change the field change event (valueChanged) to the new event (valueChange)

### PR DESCRIPTION
**Pull request by thiago-rocha**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
ci:retest

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-key-value/key_value_field.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ describe(
 						Liferay.DDM.Renderer.FieldTypes.register(
 							{
 								'name': 'key_value',
-								'templateNamespace': 'ddm.key_value'
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-key-value/key_value_field.js]: Can you order the object properties alphabetically?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-key-value/key_value_field.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ describe(
...
+								'templateNamespace': 'ddm.key_value',
+								'javaScriptClass': 'Liferay.DDM.Renderer.KeyValue'
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-key-value/key_value_field.js]: Can you order the object properties alphabetically?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-radio/radio_field.js [hunk 3]
```diff
@@ -13,7 +13,8 @@ describe(
 						Liferay.DDM.Renderer.FieldTypes.register(
 							{
 								'name': 'radio',
-								'templateNamespace': 'ddm.radio'
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-type-radio/radio_field.js]: Can you order the object properties alphabetically?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-form-renderer/field.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+					'liferay-ddm-form-renderer-field',
+					function(A) {
+						Liferay.DDM.Renderer.FieldTypes.register(
+							{
+								'name': 'field',
+								'javaScriptClass': 'Liferay.DDM.Renderer.Field'
+							}
+						);
+
+						FieldTest = A.Component.create({
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-form-renderer/field.js]: There should be a line break between ( and  {

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-form-renderer/field.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+			}
+		);
+
+		describe(
+			'Events',
+			function() {
+				it(
+					'should fire valueChange event',
+					function(done) {
+						var field = createField();
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-test/src/testFrontend/tests/dynamic-data-mapping/dynamic-data-mapping-form-renderer/field.js]: we could use the config option to make this smaller:

`var field = createField(
    {
        after: function() { ... }
    }
)`

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
Pull request submitted to https://github.com/vicnate5/liferay-portal/pull/580. [See changes here.](https://github.com/brunobasto/liferay-portal/compare/6e7adcb~0...6e7adcb) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**